Backend/app/dashboard/provider-dashboard/src/Domain/Action/ProviderPlatform/Management/Driver.hs

```diff
@@ -315,3 +318,12 @@ getDriverSecurityDepositStatus :: (ShortId DM.Merchant -> City.City -> ApiTokenI
 getDriverSecurityDepositStatus merchantShortId opCity apiTokenInfo driverId mbServiceName = do
   checkedMerchantId <- merchantCityAccessCheck merchantShortId apiTokenInfo.merchant.shortId opCity apiTokenInfo.city
   Client.callDriverOfferBPPOperations checkedMerchantId opCity (.driverDSL.getDriverSecurityDepositStatus) driverId mbServiceName
+
+instance Kernel.Types.HideSecrets.HideSecrets [Common.DriverEncDataReq] where
```

Comment:
can you can put this instance at the same place as other instances?

Reply:
just did, can you review it?